src/components/DiamondsAndCoins.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import diamondBlue from '@/assets/diamondBlue.svg'
+import coinGold from '@/assets/coinGold.svg'
+import '@/styles/DiamondsAndCoins.css'
```

Comment:
Order you imports

Eslint package - https://www.npmjs.com/package/eslint-plugin-simple-import-sort